generators/pkg/generator/generate.go

```diff
@@ -179,6 +180,20 @@ func GeneratePipeline(github *GitHub) (*v1beta1.Pipeline, error) {
 								StringVal: "success",
 							},
 						},
+						{
+							Name: "TARGET_URL",
+							Value: v1beta1.ArrayOrString{
+								Type:      v1beta1.ParamTypeString,
+								StringVal: "https://tekon/dashboard/taskrun/log",
+							},
+						},
+						{
+							Name: "DESCRIPTION",
+							Value: v1beta1.ArrayOrString{
+								Type:      v1beta1.ParamTypeString,
+								StringVal: "test",
+							},
+						},
```

Comment:
GitHub API documentation says these are optional fields, so we should omit these until we have a meaningful value to put here.

Reply:
But if I ignore these two fields, I will run into this error in the pipelinerun:
```
invalid input params for task github-set-status: missing values for these params which have no default values: [TARGET_URL DESCRIPTION]
```